forge16/src/main/java/com/envyful/api/forge/world/UtilWorld.java

```diff
@@ -25,7 +25,7 @@ public static BlockPos getRandomPosition(World world, int radiusX, int radiusZ)
                 0,
                 (UtilRandom.randomBoolean() ? 1 : -1) * UtilRandom.randomInteger(0, radiusZ));
 
-        int y = world.getHeight(Heightmap.Type.MOTION_BLOCKING, pos.getX(), pos.getZ());
+        int y = world.getChunk(pos).getHeight(Heightmap.Type.MOTION_BLOCKING_NO_LEAVES, pos.getX(), pos.getZ());
```

Comment:
Are you sure this fixes it because `world#getHeight` is doing the exact same thing.

Also, it should be `MOTION_BLOCKING` not `MOTION_BLOCKING_NO_LEAVES` otherwise the player will spawn inside leaf blocks.